sonar-db/src/main/resources/org/sonar/db/ce/CeActivityMapper.xml

```diff
@@ -67,8 +67,11 @@
           #{cUuid}
         </foreach>
       </if>
-      <if test="query.status != null">
-        and ca.status=#{query.status}
+      <if test="query.statuses != null">
```

Comment:
be careful with H2, empty IN is not supported. I think you should test != null and !empty()